grades/README.md

```diff
@@ -127,67 +127,77 @@ and type (being ABSOLUTELY sure of what you are doing):
     FinalGrade.objects.filter(course_run__edx_course_key=edx_course_key).delete()
     CourseRunGradingStatus.objects.filter(course_run__edx_course_key=edx_course_key).delete()
 
-## How to adjust (curve) proctored exam grades
+## How to import proctored exam grades
 
-Overview: After Pearson has delivered to all the proctored exam grades, we share them with the course team. 
-It typically takes them a few weeks to adjust the grades and return them to us for upload. 
+Overview: After a proctored exam has ended, the course team will deliver a spreadsheet with the grades. They need to be imported into Micromasters.
 
 ### Checklist 
 
 This can be used for creating an issue task at the end of each exam session. Just copy and paste the source markdown:
 
-- [ ] Wait for course team to deliver adjusted grades
-- [ ] Test adjusted grades and confirm counts of new course certificates and program certificates 
-- [ ] Wait for course team to confirm counts with faculty
-- [ ] load adjusted grades
-- [ ] release exam grades to learners
-- [ ] update combined final grades
-- [ ] confirm creation of course certificates and program certificates
+- [ ] Receive final exam grades from the course team
+- [ ] Load grades
+- [ ] Tally exam grades and confirm with the course team
+- [ ] Release exam grades to learners
+- [ ] Update combined final grades
+- [ ] Update certificates
 
 Detailed explanations below:
 
-### Test edx exam grades
+### Receive final exam grades from the course team
 
-The course team will deliver edx exam grades to us via a csv file in dropbox (in order to ensure the file is 
-encrypted at rest with access control). Since adjusting grades is a complex process,
-they like us to validate their changes by calculating the number of learners who, after uploading the
-grades, will have earned 1, 2, 3, 4 and 5 course certificates and how many will have earned a program certificate. 
+The course team will deliver edx exam grades to us via a csv file in dropbox (in order to ensure the file is encrypted at rest with access control). 
 
-Once we and the course team have determined that there is a match the course team will validate the grade adjustment 
-with the faculty. After the course team has validated the grades they will email us to confirm that we can
-import the adjusted grades into the micromasters application and release them to learners. 
 
-Both steps (1. validating the number of learners who have earned different numbers of course certificates and 2. 
-receiving the email confirmation from the course team to release the grades) need to occur before importing the 
-grades into the micromasters application.
+### Load grades
 
-### Load adjusted grades
+Note that users will have usernames from the platform on which they took the exams (e.g. MITx Online). If necessary, use the social_auths to convert the usernames of the exam platform to Micromasters usernames. 
+
+Steps:
```

Comment:
We have these instructions written already in exams/README.md

Reply:
@annagav  Thanks. I've added a reference to (exams/README.md), but I've kept the instructions in place here for completeness.